.changeset/pretty-flies-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@launchpad-ui/icons': minor
```

Comment:
```suggestion
'@launchpad-ui/icons': patch
```
We follow `0.x` versioning so this is technically a patch in this case for an addition.